### PR TITLE
philadelphia-generate: Improve FIX Orchestra support

### PR DESCRIPTION
**Pull request by jvirtanen**
In addition to associating an enumeration with a primary field, make it possible to associate it with secondary fields as well. Use this feature in the FIX Orchestra input source.

### CODE REVIEW

**Review by penberg — APPROVED** [review]
I don't know the code, but based on my reading of the pull request, looks good to me.

**Comment by jheusser** [comment]
Sorry missed this somehow. But same, looks good to me but a bit hard to follow if just reading the code 👍🏽
